isso/ext/notifications.py

```diff
@@ -76,7 +76,13 @@ def __init__(self, isso):
 
         self.isso = isso
         self.conf = isso.conf.section("smtp")
-        self.public_endpoint = isso.conf.get("server", "public-endpoint") or local("host")
+        self.public_endpoint = isso.conf.get("server", "public-endpoint")
+        # rstrips potential trailing '/', without having to `repr` the `local` object.
+        if self.public_endpoint:
+            self.public_endpoint = self.public_endpoint.rstrip('/')
+        else:
+            self.public_endpoint = local("host")
```

Comment:
This is redundant. Other places will always use `isso_host_script = conf.get("server", "public-endpoint") or local.host` anyway.